basex.go

```diff
@@ -78,11 +71,24 @@ func Encode(s string) (string, error) {
 	return string(reverse(result)), nil
 }
 
-// Decode converts the alpha id to big integer
-func Decode(s string) (string, error) {
+// Encode converts the big integer to alpha id (an alphanumeric id with mixed cases)
+func Encode(s string) (string, error) {
+	//numeric validation
+	if !isValidNumeric(s) {
+		return "", errors.New("Encode string is not a valid numeric")
+	}
+
+	remaining := big.NewInt(0)
```

Comment:
Looks great to me. Merging now. We can do nitpicks like renaming variables/ remove extra spaces in the next iteration